README.md

```diff
@@ -33,7 +33,7 @@ resources:
 * `password`: *Required if `auth_method` = `password`.* Admin password.
 * `admin_key`: *Required if `auth_method` = `certs`.* Base64 encoded PEM.
 * `admin_cert`: *Required if `auth_method` = `certs`.* Base64 encoded PEM.
-* `resource_type`: *Required.* Resource type to operate upon (valid values: `deployment`, `replicationcontroller`, `job`).
+* `resource_type`: *Required.* Resource type to operate upon (valid values: `deployment`, `replicationcontroller`, `job`, `pheonix-job`).
```

Comment:
yeah I know what you mean, it's a bit off... I'll try and capture something a bit more about it in the README